src/payment/strategies/braintree/braintree-paypal-payment-strategy.spec.ts

```diff
@@ -125,6 +128,8 @@ describe('BraintreePaypalPaymentStrategy', () => {
         let orderRequestBody: OrderRequestBody;
         let options: PaymentInitializeOptions;
 
+        const shippingAddress: BraintreeAddress = mapToBraintreeAddress(getShippingAddress());
```

Comment:
Since the function is already typed, I think you can safely omit the types for this.

```suggestion
        const shippingAddress = mapToBraintreeAddress(getShippingAddress());
```